CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 2.15.9 (2020-12-18)
+
+### Feature
+
+-   **pa-select**: User can no longer type in pa-select [jCarret]
+-   **pa-select**: pa-select free-text option removed [jCarret]
```

Comment:
it's a breaking change, so it should be marked as such and version number should be 2.16.0.